inference-engine/ie_bridges/java/tests/TestsSuite.java

```diff
@@ -0,0 +1,65 @@
+import org.junit.runner.RunWith;
+import org.junit.runners.AllTests;
+
+import junit.framework.TestSuite;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.zip.*;
+
+import java.nio.file.FileSystems;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+import java.lang.Class;
+import java.net.*;
+
+import org.intel.openvino.*;
+
+@RunWith(AllTests.class)
+
+public class TestsSuite extends IETest{ 
+
+    public static TestSuite suite() {
+        TestSuite suite = new TestSuite();
+        try {
+            String dir =  new File(TestsSuite.class.getProtectionDomain().getCodeSource().getLocation().toURI()).getPath().toString();
+            
+            List<Class<?>> results = findClasses(dir);
+            results.forEach(result->suite.addTest(new junit.framework.JUnit4TestAdapter(result))); 
+
+        } catch (ClassNotFoundException e) {
+            System.out.println(e.getMessage());
+        } catch (URISyntaxException e){
+            System.out.println(e.getMessage());
+        }
+        return suite;
+    }
+
+    private static List<Class<?>> findClasses(String directory) throws ClassNotFoundException {
+        List<Class<?>> classes = new ArrayList<Class<?>>();
+        try {
+
+            ZipInputStream zip = new ZipInputStream(new FileInputStream(directory));
+            for (ZipEntry entry = zip.getNextEntry(); entry != null; entry = zip.getNextEntry()) {
+                String name = entry.getName().toString();
+                if (name.endsWith(".class") && !name.contains("$") 
+                    && !name.contains("TestsSuite.class") && !name.contains("OpenVinoTestRunner.class") && !name.contains("IETest.class")) {
+                    String className = name.replace(File.separatorChar, '.');
```

Comment:
What is `className`?

Reply:
This is the name of the class in the ```.jar``` file by which the class with tests will be found.